react-form-controls/src/index.jsx

```diff
@@ -0,0 +1,36 @@
+import React, { useState } from 'react';
+import ReactDOM from 'react-dom/client';
+
+function RegistrationForm() {
+  const [changeUser, setChangeUser] = useState('');
```

Comment:
These values aren't really 'change' variables. I suggest you just name them after the state they hold. They are typically nouns. So `username` and `password` are good names.